src/pages/test-report/test-report.ts

```diff
@@ -105,23 +113,26 @@ export class TestReportPage extends BasePageComponent {
   toggleReportOverlay(): void {
     this.displayOverlay = !this.displayOverlay;
   }
+
   ionViewDidLeave(): void {
     if (super.isIos()) {
       this.deviceProvider.disableSingleAppMode();
       this.screenOrientation.unlock();
       this.insomnia.allowSleepAgain();
     }
   }
-
   ngOnDestroy(): void {
     if (this.subscription) {
       this.subscription.unsubscribe();
     }
+
+    if (this.manoeuvresSubscription) {
```

Comment:
Should have this in `subscription` and in that case we don't need this variable at all.